src/account/AccountHistory.java

```diff
@@ -0,0 +1,22 @@
+package account;
+
+import java.time.LocalDate;
+
+public class AccountHistory {
+	public boolean transaction; // deposit -> true, withdrawal -> false
+	public boolean isSuccessful;
+	public double amount;
+	public String location;
+	public LocalDate date;
+	public LocalDate clock;
+	
```

Comment:
we should store the data of sender and receiver in order to track who is sending and who is receiving